LibreNMS/Validations/Database.php

```diff
@@ -96,16 +96,16 @@ private function checkVersion(Validator $validator)
             if (version_compare($version[0], self::MARIADB_MIN_VERSION, '<=')) {
                 $validator->fail(
```

Comment:
We should add a fatal() one. Maaany people ignore this atm.

Reply:
It does stop the web ui install with this change, not lnms migrate though.